fe/fe-core/src/main/java/org/apache/doris/transaction/DatabaseTransactionMgr.java

```diff
@@ -2627,6 +2631,10 @@ private PublishResult finishCheckQuorumReplicas(TransactionState transactionStat
                         // TODO always use the visible version because the replica version is not changed
                         long newVersion = partition.getVisibleVersion() + 1;
                         for (Replica replica : tablet.getReplicas()) {
+                            if (publishTasks.get(replica.getBackendId()) == null) {
```

Comment:
don't set errorReplicaIds here,  even no this backend's publish task,  it may still ok for this replica.

errorReplicas will update in checkReplicaContinuousVersionSucc